sources/packages/backend/libs/sims-db/src/entities/disbursement-receipts.model.ts

```diff
@@ -214,4 +214,25 @@ export class DisbursementReceipt extends RecordDataModel {
     nullable: false,
   })
   studyPeriodEndDate: string;
+
+  /**
+   * The date when the NSLSC file was produced.
+   */
+  @Column({
+    name: "file_date",
+    type: "date",
+    nullable: false,
+  })
+  fileDate: string;
+
+  /**
+   * The sequence number as produced by NSLSC report.
```

Comment:
Minor, usually, the comments from here and the SQL are in sync unless there is a reason not to.
Same for the `fileDate` comment.